test/controllers/customers_controller_test.rb

```diff
@@ -0,0 +1,51 @@
+require "test_helper"
+
+describe CustomersController do
+
+  REQUIRED_CUSTOMER_FIELDS = ["id", "name", "registered_at", "postal_code", "phone", "videos_checked_out_count"].sort
+
+  # helper method
+  def check_response(expected_type:, expected_status: :success)
+    must_respond_with expected_status
+    expect(response.header['Content-Type']).must_include 'json'
+
+    body = JSON.parse(response.body)
+    expect(body).must_be_kind_of expected_type
+    return body
+  end
+
+  describe "index" do
+    it "responds with JSON and success" do
+      # Act 
+      get customers_path
+      
+      # Assert
+      check_response(expected_type: Array)
+    end
```

Comment:
These tests are great! The one piece of feedback I'd give is that this test is unnecessary since you do all these things (including the assertions) in the other two tests.